app/lib/cms/sts/assume_role_web_identity.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+module CMS
+  module STS
+    class AssumeRoleWebIdentity
+      include Singleton
+      # :reek:Attribute
```

Comment:
Is it worth adding a comment here?

Reply:
Changed in 18c48a2.

Reply:
This magic comment was removed in c89d43e :v: